src/main/java/com/example/campus/repository/AnswerRepository.java

```diff
@@ -9,8 +9,20 @@
 @Mapper
 public interface AnswerRepository {
 
+	/**
+	 * 全件取得
+	 * @param 検索結果
+	 */
 	public List<Answers> findAll();
 
+	/**
+	 * 新規作成
+	 */
 	public void create(Answers answer);
 
+	/**
+	 * 更新
+	 */
+	public Answers update(Answers answer);
```

Comment:
更新なので返り値は無しでいいと思います。
```suggestion
	public void update(Answers answer);
```